src/app/story/storysidebar/storysidebar.component.css

```diff
@@ -19,34 +29,43 @@
     width: 25px;
 }
 
-#sidebarTitle {
-    border-bottom: 2px solid #A9A9A9;
-    font-size: 1.5rem;
-    font-weight: bold;
-    padding: 0.75rem;
+#sidebar.closed > * {
+    /* When the sidebar is closed, make the arrow closer to the edge */
+    margin-left: 0.75rem;
+}
+
+#sidebar > * {
+    /* Move all content in the sidebar over 2.5rem to not hug the border */
+    margin-left: 2.5rem;
```

Comment:
I just want to confirm that 'rem' will be suitable for all screen sizes and resolutions?

Reply:
It looked good from what I could tell. We can adjust if we get feedback.